source/conf.py

```diff
@@ -160,5 +161,11 @@
      'Miscellaneous'),
 ]
 
+intersphinx_mapping = {
+    'community': (
+        f"https://www.tarantool.io/{language}/doc/latest",
```

Comment:
language will never change in this projectd

```suggestion
        f"https://www.tarantool.io/en/doc/latest",
```